core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -165,6 +169,8 @@
         super("Install a plugin");
         this.batchOption = parser.acceptsAll(Arrays.asList("b", "batch"),
                 "Enable batch mode explicitly, automatic confirmation of security permission");
+        this.quietOption = parser.acceptsAll(Arrays.asList("q", "quiet"),
```

Comment:
We already have the `silent` option on the parent Command class, why do we need this too?